src/host/input.cpp

```diff
@@ -347,20 +344,13 @@ void ProcessCtrlEvents()
         if (NT_SUCCESS(Status))
         {
             Status = ServiceLocator::LocateConsoleControl()
-                         ->EndTask(rgProcessHandleList[i].dwProcessID,
+                         ->EndTask(r.dwProcessID,
                                    EventType,
                                    CtrlFlags);
-            if (rgProcessHandleList[i].hProcess == nullptr)
+            if (!r.hProcess)
             {
                 Status = STATUS_SUCCESS;
             }
         }
-
-        if (rgProcessHandleList[i].hProcess != nullptr)
-        {
-            CloseHandle(rgProcessHandleList[i].hProcess);
-        }

```

Comment:
Don't we still need this in the event that `if (NT_SUCCESS(Status))` fails?

Reply:
I made it so that the `hProcess` is a `wil::unique_handle` so it'll get cleaned up automatically.